transfer.py

```diff
@@ -0,0 +1,214 @@
+import csv
+from datetime import datetime
+from html import unescape
+import json
+import os
+import re
+from urllib.parse import unquote
+import urllib3
+
+import boto3
+import click
+from oauthlib import oauth1
+import requests
+
+# Ignore insecure requests warnings
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
I thought this warning belonged to the past with urllib3... :thinking:

Reply:
well... it does hide all the warnings when I add this.

Reply:
Yes I understand, but I thought that this warning would be fixed with python 3. In fact, I think it's a weird binding with an old openssl library that is installed in the base image.